pkg/ekcoops/operator.go

```diff
@@ -221,3 +228,32 @@ func shouldUseNodeForStorage(node corev1.Node, rookStorageNodesLabel string) boo
 	}
 	return false
 }
+
+//TODO approve the csrs if the SIGNERNAME is kubernetes.io/kubelet-serving
+
+func (o *Operator) reconcileCertificateSigningRequests() error {
+	csrList, err := o.client.CertificatesV1().CertificateSigningRequests().List(context.TODO(), metav1.ListOptions{})
+	if err != nil {
+		if kuberneteserrors.IsNotFound(err) {
```

Comment:
Is an is not found error possible from a list call?

Reply:
not possible :) thanks